src/lib/core/CHIPConfig.h

```diff
@@ -1339,3 +1358,14 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 /**
  * @}
  */
+
+/**
+ *  @def CHIP_CONFIG_PERSIST_SUBSCRIPTIONS
+ *
+ *  @brief
+ *    If asserted (1), suppress definition of the standard error formatting function
```

Comment:
This does not seem correct description
Also, since this feature looks quite heavy and, as far as I understand, optional (especially, in the context of Client Monitoring cluster), would it make sense to disable it by default and only enable it for selected platforms?

Reply:
Thank you for catching the incorrect description. I'll fix that in my next patch.

This feature is optional, and we will make a separate PR before 1.1 ships to change it to 0. This is currently defined as 1 because we want this to be tested in CI during development.